components/HackathonModal.tsx

```diff
@@ -0,0 +1,94 @@
+// What is this?
+// For our hackathon, we wanted to have a extra way to display and grab the user's attention
```

Comment:
```suggestion
// This modal is a temporary modal for the Trent Hacks 2024 Hackathon.
```

I think its important we note the importance and specififty of it.